### ISSUE TITLE
Feature: Multiple transaction description

### ISSUE THREAD

**Issue by corebyte** [WoLpH/mt940#9]
Now the library reads the last line of :86: description multi lines as data['transaction_details'].

Would it be possible to parse all the :86: lines from a transaction and put them in an array or some sort in the parameter transaction_details?

That would be really helpful.

**Comment by nms84** [comment]
It reads all lines for me. Perhaps you're on the wrong version? The TransactionDetails class in 'tags.py' should look like this:

```python
class TransactionDetails(Tag):
    '''Transaction details

    Pattern: 6x65x
    '''
    id = 86
    scope = mt940.models.Transaction
    pattern = r'(?P<transaction_details>[\s\S]{0,330})'
```

**Comment by WoLpH** [comment]
It should put it all in one string, to get an array simply do a split like this:

    transactions_details.split('\n')

**Comment by corebyte** [comment]
Sorry, i mean the structured file.

I tried importing my Rabobank MT940 structured transaction file. It works all out of the box. Except the description, it is splitted on multiple :86: rules. If that works then it is also supported for structured.

Because MT940 is deprecated and structured is the new language for importing transactions.

**Comment by WoLpH** [comment]
Ah... right, Rabobank appears to be doing things differently from the other banks. I'll write a patch to automatically combine multiple similar tags together.

**Comment by WoLpH** [comment]
Fixed in Version 2.2 :)
https://github.com/WoLpH/mt940/commit/d926428d456994fb1f341534042bdd03a02a023a

**Comment by corebyte** [comment]
Thanks for the good work @WoLpH!
